src/coreComponents/constitutive/solid/ElasticIsotropic.cpp

```diff
@@ -58,6 +60,15 @@ ElasticIsotropic::ElasticIsotropic( string const & name, Group * const parent ):
   registerWrapper( viewKeyStruct::shearModulusString(), &m_shearModulus ).
     setApplyDefaultValue( -1 ).
     setDescription( "Elastic Shear Modulus Field" );
+
+  registerWrapper( viewKeyStruct::thermalExpansionCoefficientString(), &m_thermalExpansionCoefficient ).
+    setApplyDefaultValue( -1 ).
+    setDescription( "Thermal Expansion Coefficient Field" );
+
+  registerWrapper( viewKeyStruct::defaultThermalExpansionCoefficientString(), &m_defaultThermalExpansionCoefficient ).
+    setApplyDefaultValue( 0.0 ).
```

Comment:
Yes, this is consistent now, both have a default value of `0.0`. For now it does not matter because the default value of `thermalExpansionCoefficient` is never used, but in case it changes in the future, they now have the same value.